AudioManagerUtilities/src/CAmSocketHandler.cpp

```diff
@@ -1124,7 +1127,6 @@ bool CAmSocketHandler::nextHandle(sh_identifier_s & handle)
         }
         if (handle.lastUsedID == lastHandle)
         {
-            logError("Could not create new polls, too many open!");
             return (false);
```

Comment:
Why not worth logging?  At least comment...

Reply:
The rest looks good to me.

Reply:
The logging is done in all cases where the function gets called. That is why I have removed it.

Reply:
Right, the log message was completely wrong.  I see that now.